frontend/src/app/main/data/workspace/bool.cljs

```diff
@@ -99,7 +100,10 @@
              shapes (->> ordered-indexes
                          (map (d/getf objects))
                          (remove cph/frame-shape?)
-                         (remove #(ctn/has-any-copy-parent? objects %)))]
+                         (remove #(ctn/has-any-copy-parent? objects %))
+                         (remove #(->> %
+                                       (get objects)
```

Comment:
Is this correct, i mean, on the line 101 you already resolve the id to shape, so this `get  objects` looks redundant and incorrect